analysers/experiments/numberOfConstructInstances.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import scipy.integrate as integrate
+import scipy.special as special
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(dataRDD, experimentID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    combinations = dataRDD.map(lambda a: (a["construct_type"], a["construct_name"])).distinct().collect()
+    
+    for combs in combinations:
+        mode = computeMode(dataRDD.filter(lambda r: r['construct_name'] == combs[1] and r['construct_type'] == combs[0]).map(lambda r: (r['number_of_construct_instances'], 1)))
```

Comment:
This is often present in the code: why do you have two very similar `filter` + `map` here and on line https://github.com/benchflow/analysers/pull/75/files#diff-ff083bb8138c0743d316956959d94012R28?